### DISCUSSION TITLE
Update-2023-08-11 Teaser Trailer 3

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#242] [Announcements]
Something fun for the weekend: another cheesy teaser trailer. This time showcasing the recent particle and sprite features.

https://youtu.be/liaLgaTOpYE

Enjoy! :)

**Comment by kayomarz** [comment]
I like the effect which you have towards the end of the trailer... looks like a moving wheel letting off some smoke:) looks cool!

**Reply by kaveh808** [reply]
That is Demo 08 in `demo-sprite.lisp`.
